src/category_theory/triangulated/basic.lean

```diff
@@ -29,13 +29,12 @@ open category_theory.category
 /--
 We work in an additive category C equipped with an additive shift.
 -/
-variables (C : Type u) [category.{v} C] [has_shift C] [additive_category C]
-[functor.additive (shift C).functor]
-
+variables (C : Type u) [category.{v} C] [additive_category C]
+variables [has_shift C] [functor.additive (shift C).functor]
```

Comment:
Is there a reason to repeat `variables` here?

Reply:
No, that was probably a bad change, sorry. I originally intended just to add an indent on the 2nd line, but then added `variables` instead.